src/main/java/com/compilercharisma/chameleonbusinessstudio/config/SecurityConfiguration.java

```diff
@@ -20,6 +20,7 @@ public SecurityWebFilterChain securityWebFilterChain(
                 .oauth2Login()
                 .and()
                 .authorizeExchange()
+                .pathMatchers(HttpMethod.POST,   "/api/v1/users").authenticated()                // this will be used for the log in flow after a user is authenticated
```

Comment:
Ah so that's what `authenticated()` does. Nice.

Reply:
Yeeee, it checks if the user authenticated using whichever method of authentication we chose, it is very abstract so in this case, we are using OAuth2